tideways/provision.sh

```diff
@@ -74,6 +74,28 @@ install_xhgui() {
     fi
 }
 
+enable_tideways_by_site() {
+    rm /srv/tideways.json
+
+    domains=""
+    input=$(cat "${VVV_CONFIG}")
+    sites=$(echo "$input" | shyaml keys-0 sites | xargs -0 -n 1 echo "")
+
+    for value in ${sites}; do
+        domain=$(echo "$value" | tr -d '[:space:]')
+        tideways=$(echo "$input" | shyaml -q get-value sites."$domain".tideways)
+        if [[ $tideways != "" ]]; then
+            hosts=$(echo "$input" | shyaml -q get-value sites."$domain".hosts)
+            domains="$domains${hosts/\- /''}\n"
+        fi
+    done
+    SAVEIFS=$IFS   # Save current IFS
+    IFS=$'\n'      # Change IFS to new line
+    domains=("$domains")
+    IFS=$SAVEIFS   # Restore IFS
+    printf "${domains[@]}" | jq -R . | jq -s . > /srv/tideways.json
```

Comment:
Do we have `jq` available by default?

Reply:
Also, I'd like to be able to provision with no domains or sites in the config file at all for box building purposes, so a check if that's the case would be good.

There's probably another way of grabbing hosts too that would allow `vvv_hosts` entries to be added. I forget if these are present in the hosts file or not inside the VM, if so the full list can be gotten from there, then compared against the VVV config for enabling/disabling

Reply:
seems that jq is not installed in VVV but I think that is better to add it because is the only way to manage easily json files in bash.
Right now to enable tideways if the domain is enabled by config has priority over the parameter in the url. So we can keep both the features.

Reply:
Would it be better to use PHP for that? Or `node` even?

Reply:
Well have `jq` means that we can use it also elsewhere in VVV.
Anyway write a script in php that if called from a web server (like dashboard) append the site and on CLI do the scan of the `yaml` can be a way to have a unique codebase.